Automattic/woocommerce-payments
# [Simplifying Deposits UI] Update the deposit status chip to use our new WC Pay Pill

> **Warning** - This PR is based off #5890 which should be merged before this branch

Fixes #5892

#### Changes proposed in this Pull Request









In https://github.com/Automattic/woocommerce-payments/pull/5890 we updated our WCPay pill component to use of the [WC Core `<Pill>`](https://woocommerce.github.io/woocommerce-admin/components/storybook/index.html?path=/story/woocommerce-admin-components-pill--default) component, in this PR I'm updating our `<DepositStatusChip />` to use our new Pill component and map the given deposit status to the correct pill type for styling and correct status label.

The changes coming in this PR include:
 - Rename the component to `DepositStatusPill`.
 - Given a deposit status, map the status to the Pill type and render the pill label
 - Update the existing `components/deposits-overview/next-deposit` component to use the new deposit status pill.

**Usage**

```
<DepositStatusPill status="estimated" />
<DepositStatusPill status="pending" />
<DepositStatusPill status="in_transit" />
<DepositStatusPill status="paid" />
<DepositStatusPill status="failed" />
<DepositStatusPill status="canceled" />
```

**Output**

![image](https://user-images.githubusercontent.com/2275145/229406664-f4c4856e-c8e2-42b1-86ef-60b30c2bd8ea.png)


#### Testing instructions





1. Checkout this branch and run `npm run build:client`
2. Set `_wcpay_feature_simplify_deposits_ui` to `1` in wp-options table
3. Go to the Payments > Overview page
4. Confirm the Next Deposit table has an estimated deposits status pill styled correctly
5. Since we don't have any other deposit statuses using this pill you can use the above code to render all of the deposit pills

-------------------


**Post merge**
